alphastats/DataSet_Preprocess.py

```diff
@@ -31,10 +71,6 @@ def _subset(self):
         )
         return self.mat[self.mat.index.isin(self.metadata[self.sample].tolist())]
 
-    def preprocess_print_info(self):
```

Comment:
This is used in the MQ tutorial, so we should replace it there, if we remove it.

Reply:
is it? I couldn't find it anywhere?

Reply:
It's an html file, so maybe only in the docs?

Reply:
removed it also from here https://github.com/MannLabs/alphapeptstats/pull/324/files#diff-46145cbe5211ac920a0dc16d2c095e92dd4730da4dbe6e58776f1b3d407cde09